Community-Supported/hyper-jupyter-kernel/README.md

```diff
@@ -0,0 +1,141 @@
+
+Being a Jupyter kernel, the `hyper_kernel` can be used in combination with the rest of the Jupyter ecosystem.
+In particular:
+* It can be run from an old-school or over a ssh-connection using `jupyter console --kernel=hyper`
+* It can be used from JupyterLab's consoles, providing a more modern UI experience
+* It can be used with JupyterLab notebooks, allowing to tell stories based on your SQL queries
+
+## Get started using JupyterLab
+
+JupyterLab is a web-based IDE for data-scientist.
+With the `hyper_kernel` you can integrate HyperAPI's SQL interface directly into this IDE.
+
+![Screenshot of jupyter console](media/jupyterlab.png)
+
+First, install jupyter-console and the kernel provided in this repository using
+
+```
+python -m pip install jupyter-console tableauhyperapi git+https://github.com/tableau/hyper-api-samples#subdirectory=Community-Supported/hyper-jupyter-kernel
+python -m hyper_kernel.install
+```
+
+Next launch JupyterLab using
+
+```
+python -m jupyter lab
+```
+
+and open JupyterLab in your browser
+
+
+## Get started using the command-line interface
+
+![Screenshot of jupyter console](media/jupyter_console.png)
+
+First, install jupyter-console and the kernel provided in this repository using
+
+```
+python -m pip install jupyter-console tableauhyperapi git+https://github.com/tableau/hyper-api-samples#subdirectory=Community-Supported/hyper-jupyter-kernel
+python -m hyper_kernel.install
+```
+
+Afterwards, you can launch the console using
+
+```
+jupyter console --kernel=hyper
+```
+
+## Running your first query
+
+Let's start with something simple: Enter
+
+```
+SELECT 12/2 AS result
+```
+
+into your console and submit it. You should get `6` as a result. Congrats! You just executed your first query.
+
+# Quick start for common use cases
+
+Independent of how you run your `hyper_kernel`, the usage is always similar:
+You feed SQL to it, and in return you get the query results.
+The [HyperAPI SQL reference](https://help.tableau.com/current/api/hyper_api/en-us/reference/sql/index.html) contains all the information you need.
+For everyone who prefers to not read the whole reference first, here is a short quick start on the most common scenarios:
+
+
+## Creating a Hyper database
+
+That was boring, though... Instead of only using Hyper as your calculator, you usually want to use it to crunch some data.
+
+First let's [create a new database](https://help.tableau.com/current/api/hyper_api/en-us/reference/sql/sql-createdatabase.html):
+```
+CREATE DATABASE "my_data.hyper";
+```
+
+and load that newly created database into our session:
+```
+ATTACH DATABASE "my_data.hyper" AS database1;
+```
+
+Next, we will need to [create a table](https://help.tableau.com/current/api/hyper_api/en-us/reference/sql/sql-createtable.html):
+```
+CREATE TABLE database1.public.my_table (column1 text, column2 bigint);
```

Comment:
Do we need to explicitly name `database1.public`? If yes, can you please add a comment why we need to name it that way, and what `public` is and why we need it there.

Reply:
no, we don't need to name it explicitly, we could also just write "my_table" here.

However, it tells a more coherent story if the identifier given to ATTACH also appears again in later commands

Reply:
I would either just use "my_table" or give a short explanation about our naming

Reply:
> I would either just use "my_table"
I still disagree. Please provide a rational, otherwise I can only treat this as your personal opinion - and it's hard to argue about personal opinions...

> give a short explanation about our naming
I would like to keep this document as short as possible. I already linked our SQL reference, which contains the details about our SQL functionality

Reply:
The argument mainly is: The syntax databasealias.schemaname.tableaname is just not common knowledge and would possibly confuse ppl trying to understand what's going on.

Reply:
Added a small explanation of the `database1` parameter